docs/modules/ROOT/pages/sentinel.adoc

```diff
@@ -0,0 +1,211 @@
+[[sentinel]]
+= Sentinel
+
+_The Defender Sentinel service allows you to monitor transactions for Addresses by specfying Events, Functions, and Transaction conditions and notifying via email, slack, and autotasks (more to come)._
+
+[[use-cases]]
+== Use cases
+
+Use a Sentinel whenever you need to know about or respond to transactions involving your smart contract.  Your Sentinel will watch every transaction send the transactions you care about to your notification methods of choice.
+
+* *Monitor* your *sensitive functions* like transferOwnership, pause
+* *Alert* when transactions are failing against your smart contract
+* *Respond* to transactions by executing *Autotask* logic when key events happen 
+* *Integrate* with your *existing tools* through slack, email or custom *Autotask* integration
+* *Know* when an unexpected volume of transactions occur
+
+[[whats-in-an-sentinel]]
+== What's in an Sentinel?
+
+A Sentinel watches transactions for a given address, filters those transactions by any conditions that you specify, then notifies you when those transactions occur.
+
+IMPORTANT: At this time, Sentinels do not monitor internal function calls.
+
+[[specify-address]]
+=== Specifying an Address
+
+* *Address*: Choose the address of either a Smart Contract or a normal key-pair address to monitor.
+
+* *ABI*: Specify the ABI for your contract.  We will attempt auto-discovery of the ABI if it is verified.
+
+* *Confirmation Blocks*: If you want to be notified only after a certain level of confidence that the transaction is accepted, choose a higher confirmation block level.   If you want to be notified as soon as possible and are tolerant to re-orgs, then choose *Latest* 
+
+[[matching-rules]]
+=== Matching Rules
+
+For a Transaction to trigger a notification, it must satisfy *ALL* of the following:
+
+* Transaction *MUST* have a To, From, or Address (from log) that matches the configured address
```

Comment:
```suggestion
* Transaction *MUST* have a To, From, or an Event that matches the configured address
```

Reply:
Also, and this is personal, I'd drop the RFC-like notation of **MUST** and similar